platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/MarkerViewManager.java

```diff
@@ -31,27 +32,33 @@
  */
 public class MarkerViewManager {
 
-    private Map<MarkerView, View> markerViewMap;
+    private final ViewGroup markerViewContainer;
+    private final Map<MarkerView, View> markerViewMap;
+    private final List<MapboxMap.MarkerViewAdapter> markerViewAdapters;
+
+    // TODO refactor MapboxMap out for Projection and Transform
+    // Requires removing MapboxMap from Annotations by using Peer model from #6912
     private MapboxMap mapboxMap;
-    private MapView mapView;
-    private List<MapboxMap.MarkerViewAdapter> markerViewAdapters;
+
     private long viewMarkerBoundsUpdateTime;
     private MapboxMap.OnMarkerViewClickListener onMarkerViewClickListener;
-    private ImageMarkerViewAdapter defaultMarkerViewAdapter;
 
     /**
      * Creates an instance of MarkerViewManager.
      *
-     * @param mapboxMap the MapboxMap associated with the MarkerViewManager
-     * @param mapView   the MapView associated with the MarkerViewManager
+     * @param container the ViewGroup associated with the MarkerViewManager
      */
-    public MarkerViewManager(@NonNull MapboxMap mapboxMap, @NonNull MapView mapView) {
-        this.mapboxMap = mapboxMap;
+    public MarkerViewManager(@NonNull ViewGroup container) {
+        this.markerViewContainer = container;
         this.markerViewAdapters = new ArrayList<>();
```

Comment:
The initializations that have no dependencies might as well be done in the field declaration. That also enables us to make them final and maybe skip some null checks here and there.